ProjectSettings/EditorBuildSettings.asset

```diff
@@ -4,4 +4,8 @@
 EditorBuildSettings:
   m_ObjectHideFlags: 0
   serializedVersion: 2
-  m_Scenes: []
+  m_Scenes:
```

Comment:
@jwittner @dbouianov looks like a few things got missed in code review?
Please update this file to it's original settings.